### ISSUE TITLE
Generate URLs for files on Crowdin so we can link to them from the website

### ISSUE THREAD

**Issue by zeke** [electron/electron-i18n#133]
From Andriy at Crowdin:

> You may try to use Project Details endpoint:
> https://support.crowdin.com/api/info/
? it dispays information about paths and files ID

> Response sample:

```  <files> <!-- project files details -->
    <item>
    <node_type>directory</node_type>
    <id>1</id>
    <name>test</name>
    <files>
      <item>
        <node_type>file</node_type>
        <id>2</id>
        <name>market-info.txt</name>
        <created>2017-05-19T13:08:13+0000</created>
        <last_updated>2017-05-20T18:02:13+0000</last_updated>
        <last_accessed>2017-05-21T09:10:48+0000</last_accessed>
        <last_revision>1</last_revision>
      </item>
```

> It means, there is a directory named "a" containing file named "market-info.txt" and its ID is *2*

The API supports JSON responses too, so we can do something like this:

```
const url = `https://api.crowdin.com/api/project/electron/info?key=${process.env.CROWDIN_KEY}&json`
```

See `script/stats.js` for prior art.

**Comment by zeke** [comment]
A step: https://github.com/zeke/get-crowdin-file-ids#readme

**Comment by zeke** [comment]
Landed in #230
